python/lsst/scarlet/lite/models/free_form.py

```diff
@@ -126,10 +130,119 @@ def resize(self, model_box: Box) -> bool:
 
     def __str__(self):
         return (
-            f"FreeFormComponent(\n    bands={self.bands}\n    "
+            f"FactorizedFreeFormComponent(\n    bands={self.bands}\n    "
             f"spectrum={self.spectrum})\n    center={self.peak}\n    "
             f"morph_shape={self.morph.shape}"
         )
 
     def __repr__(self):
         return self.__str__()
+
+
+class FreeFormComponent(Component):
+    """Implements a free-form component
+
+    This is a FreeFormComponent that is not factorized into a
+    spectrum and morphology with no monotonicity constraint.
+    """
+
+    def __init__(
+        self,
+        bands: tuple,
+        model: np.ndarray | Parameter,
+        model_bbox: Box,
+        bg_thresh: float | None = None,
+        bg_rms: np.ndarray | None = None,
+        floor: float = 1e-20,
+        peaks: list[tuple[int, int]] | None = None,
+        min_area: float = 0,
+    ):
+        if len(bands) != 1:
+            raise ValueError("MonochromaticDeconvolvedComponent only supports one band")
+        super().__init__(bands=bands, bbox=model_bbox)
+        self._model = parameter(model)
+        self.bg_rms = bg_rms
+        self.bg_thresh = bg_thresh
+        self.floor = floor
+        self.peaks = peaks
+        self.min_area = min_area
+
+    @property
+    def model(self) -> np.ndarray:
+        """The morphological model of the component"""
```

Comment:
If the base class has full docstrings, which it should, is there any point to adding these?